### DISCUSSION TITLE
AreTomo tilt angle off by 90˚?

### DISCUSSION THREAD

**Discussion by asarnow** [nextpyp/.github#105] [Q&A] [answered]
AFAICT the user specified tilt axis is just passed unaltered to AreTomo - am I mistaken? If the angle in the nextPYP UI is 175˚ then AreTomo needs to get 85˚ on the command line, which I think is the same situation as for IMOD. Is the user tilt angle used for anything other than passing to IMOD/AreTomo? Passing 85˚ to nextPYP as a workaround doesn't seem to give the expected result.

**Comment by tofushan** [comment]
The convention in the UI is different from the ones used by IMOD and AreTomo, so the value from UI will be subtracted by 90 before passing it to IMOD or AreTomo. This is answered in #21 .

**Comment by abartesaghi** [comment, marked as answer]
@asarnow, you were right, the tilt-axis angle passed to AreTomo2 was the same angle specified in nextPYP's UI. 

Starting with version [0.6.5](https://nextpyp.app/files/pyp/0.6.5/docs/index.html), the tilt-axis angle passed to AreTomo2 and IMOD are both obtained by subtracting 90 degrees from the angle specified in nextPYP's UI.

Regarding your second question, the tilt-axis orientation is only used for the purpose of aligning the tilt-series.

**Reply by asarnow** [reply]
Thanks for investigating! I would have posted a proof-of-bug but I was on travel.
